Vagrantfile

```diff
@@ -33,6 +33,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
   end
 
+  for forwarded_port in vconfig['vagrant_forwarded_ports']
```

Comment:
Can you wrap this entire set of config in an `if` statement and check if `vagrant_forwarded_ports` is a key in the `vconfig` array? For those updating, I don't want `vagrant up` to fail if the variable is not defined in `config.yml`.

And this would allow us more flexibility in terms of not having it in the default config.yml, but still available if someone wants it.